pkg/pillar/cmd/volumemgr/handleconfig.go

```diff
@@ -79,6 +80,13 @@ func vcCreate(ctx *volumemgrContext, objType string, key string,
 				log.Error(errStr)
 				initStatus.SetError(errStr, time.Now())
 			} else {
+				info, err := os.Stat(ociFilename)
+				if err != nil {
+					errStr := fmt.Sprintf("Calculating size of container image failed: %v", err)
+					log.Error(errStr)
+				} else {
+					dos.MaxSizeBytes = uint64(info.Size())
```

Comment:
You shouldn't need the MaxSizeBytes to do the snapshot.
We should track the actual size separately from the MaxSize; don't we have a field for this in VolumeStatus?

Reply:
Currently, we have two fields ```TargetSizeBytes``` and ```MaxSizeBytes``` in ```DownloadConfigOrigin``` in the VolumeStatus.
For container applications, both sizes are the same. And while creating container snapshots, we are not using sizes anywhere. It is only used for reporting purposes.

Reply:
But if we want dos.MaxSizeBytes to be the size what we actually downloaded (as opposed to what we got from the MaxSizeBytes in the config), then we should apply that across the board and not just for containers.

Reply:
With these changes, we are applying it for both VM and containers.

Reply:
OK